Goldlight.HttpClientTestSupport/FakeHttpMessageHandler.cs

```diff
@@ -418,6 +418,18 @@ public FakeHttpMessageHandler WithExpectedContent(string content)
       return this;
     }
 
+    /// <summary>
+    /// Set the content that is expected in the response.
+    /// </summary>
+    /// <param name="content">The content to populate the response.</param>
+    /// <param name="options">Options for custom serialization of the content.</param>
+    public FakeHttpMessageHandler WithExpectedContent<T>(T content, JsonSerializerOptions options) where T : class
```

Comment:
Be worth defaulting the options so the user doesn't need to specify the JsonSerializerOptions if they don't have to, i.e., it sets the options to new JsonSerializerOptions if one isn't passed in.

Reply:
There is an overload that only takes `T content` and no options. Would you prefer a single method with defaulted options?

Reply:
I was thinking of a single method with defaulted options.

Reply:
Updated